scripts/scil_apply_transform_to_tractogram.py

```diff
@@ -2,13 +2,22 @@
 # -*- coding: utf-8 -*-
 
 """
-    Transform tractogram using an affine/rigid transformation.
-
-    For more information on how to use the various registration scripts
-    see the doc/tractogram_registration.md readme file
+Transform tractogram using an affine/rigid transformation.
+
+For more information on how to use the various registration scripts
+see the doc/tractogram_registration.md readme file
+
+Applying transformation to tractogram can lead to invalid streamlines (out of
+the bbox), three strategies are available:
+- default, crash at saving if invalid streamlines are present
+- --keep_invalid, save invalid streamlines. Leave it to the user to run
+    scil_remove_invalid_streamlines.py if needed.
+- --remove_invalid, automatically remove invalid streamlines before saving.
+    Should not remove more than a few streamlines.
```

Comment:
Please use exact same wording in both scripts. scil_apply_warp uses a bit different wording. Please use the exact same to avoid confusion.